tests/src/performance/Scenario/JitBench/JitBenchHarness.cs

```diff
@@ -135,115 +135,91 @@ private static void SetupStatics(JitBenchHarnessOptions options)
             s_jitBenchDevDirectory = Path.Combine(s_temporaryDirectory, "J");
             s_dotnetProcessFileName = Path.Combine(s_jitBenchDevDirectory, ".dotnet", "dotnet.exe");
             s_musicStoreDirectory = Path.Combine(s_jitBenchDevDirectory, "src", "MusicStore");
+
+            s_localJitBenchRepo = options.LocalJitBenchRepo;
         }
 
         private static void DownloadAndExtractJitBenchRepo()
         {
-            using (var client = new HttpClient())
-            {
-                var archiveName = $"{JitBenchCommitSha1Id}.zip";
-                var url = $"{JitBenchRepoUrl}/archive/{archiveName}";
-                var zipFile = Path.Combine(s_temporaryDirectory, archiveName);
+            // If the repo already exists, we delete it and extract it again.
+            if (Directory.Exists(s_jitBenchDevDirectory))
+                Directory.Delete(s_jitBenchDevDirectory, true);
 
-                using (FileStream tmpzip = File.Create(zipFile))
+            if (s_localJitBenchRepo == null)
+            {
+                using (var client = new HttpClient())
                 {
-                    using (Stream stream = client.GetStreamAsync(url).Result)
-                        stream.CopyTo(tmpzip);
-                    tmpzip.Flush();
-                }
+                    var archiveName = $"{JitBenchCommitSha1Id}.zip";
+                    var url = $"{JitBenchRepoUrl}/archive/{archiveName}";
+                    var zipFile = Path.Combine(s_temporaryDirectory, archiveName);
 
-                // If the repo already exists, we delete it and extract it again.
-                if (Directory.Exists(s_jitBenchDevDirectory))
-                    Directory.Delete(s_jitBenchDevDirectory, true);
+                    using (FileStream tmpzip = File.Create(zipFile))
+                    {
+                        using (Stream stream = client.GetStreamAsync(url).Result)
+                            stream.CopyTo(tmpzip);
+                        tmpzip.Flush();
+                    }
 
-                // This step will create s_JitBenchDevDirectory.
-                ZipFile.ExtractToDirectory(zipFile, s_temporaryDirectory);
-                Directory.Move(Path.Combine(s_temporaryDirectory, $"JitBench-{JitBenchCommitSha1Id}"), s_jitBenchDevDirectory);
+                    // This step will create s_JitBenchDevDirectory.
+                    ZipFile.ExtractToDirectory(zipFile, s_temporaryDirectory);
+                    if (s_localJitBenchRepo == null)
```

Comment:
This code block is executed when `s_localJitBenchRepo == null`. This check is not needed here.